book/modules/using_modules.md

```diff
@@ -69,25 +69,39 @@ The path to the module can be:
   Note that the module name (its directory) can end in a `/` (or `\` on Windows), but as with most commands that take a paths (e.g., `cd`), this is completely optional.
   :::
 
-  ::: important Important! Importing modules from `$env.NU_LIB_PATH`
+  ::: important Important! Importing modules from `$env.NU_LIB_DIRS`
   When importing a module via a relative path, Nushell first searches from the current directory. If a matching module is not found at that location, Nushell then searches each directory in the `$env.NU_LIB_DIRS` list.
 
   This allows you to install modules to a location that is easily accessible via a relative path regardless of the current directory.
   :::
 
-- An absolute or relative path to a Nushell module file. As above, Nushell will search the `$env.NU_LIB_DIRS` for a matching relative path.
+- An absolute path to a Nushell module file:
```

Comment:
Before the fixup, the first two was split and the second was combined. I think that asymmetry should be solved, one way or another.

I agree with order should be relative before absolute

I prefer self-sufficient explanations, without pointing to earlier sections. I would prefer duplication over pointing elsewhere, especially if it's not a anchor jump link

Given the noteworthy concerns, maybe it makes more sense to combine the two absolute and the two relative blocks? 🤔

Reply:
> Before the fixup, the first two was split and the second was combined
> Given the noteworthy concerns, maybe it makes more sense to combine the two absolute and the two relative blocks?

Ah - I think I had a good reason for it, but I also agree with you on the symmetry.  I'll take another look at it, but it might be a few hours.

Reply:
I'm fine with leaving it as it was. Personally, I like reference documentation. But this is a "book". So I'm reminding myself that the form is more of a read-in-order thing where reading order and block-cross-references can be expected. Unfortunately, there's no reference then though. :)